geemusic/intents/playback.py

```diff
@@ -81,6 +82,25 @@ def prev_song():
 
         return audio().play(stream_url)
 
+@ask.intent("AMAZON.ShuffleOnIntent")
+def shuffle_tracks():
+    shuffled_songs = queue.song_ids
+    
+    if shuffled_songs == [] or None:
+        return statement("There are no songs to shuffle.")
+    else:
+        api = GMusicWrapper.generate_api()
+    
+        # This will randomly reorder the song ids thereby "shuffling" the tracks
+        random.shuffle(shuffled_songs)
```

Comment:
Why do we need to modify the list read-only. We will be replaying `song_ids` from the beginning and `random.shuffle()` has the bonus of being easier to understand by someone modifying the code base. The underlying code for the randomization [is the same](http://stackoverflow.com/questions/12798363/what-is-the-difference-between-random-sample-and-random-shuffle-in-python)